README.md

```diff
@@ -34,6 +34,24 @@ The barcode column can take values following the format barcode[0-9][0-9] or NB[
 
 Note: Basecalling and demultiplexing may also be performed if not already done on GridION/MinIT.
 
+## Basic usage using Dockerfile/image
+
+The Dockerfile/image is made to be used with Docker-out-of-Docker (DooD), where we mount `/var/run/docker.sock` into our container and our container will start new containers that are "siblings" to the one we started it from, and thus runs them in the context of the host. Due to this we have to use absolute paths when mounting volumes and setting `--input_dir` and `--sample_names`. 
+
+Assuming you have the sample_names.csv file in your `<run_name>` folder:
+
+``` shell
+docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v /absolute/path/to/<run_name>:/absolute/path/to/<run_name>
+cov:test python susCovONT.py --yes --user_id 0 --input_dir /absolute/path/to/<run_name> --sample_names /absolute/path/to/<run_name>/sample_names.csv
+```
+
+Where:
+* `--yes`: Will run the pipeline non-interactively and give an automatic yes to all prompts
+* `--user_id`: User id used to run docker commands (default: 1000), here we want to use 0 for root. 
+* `--input_dir` and `--sample_names` are described above
+
+For increased reproducibility also consider specifying the Nextclade version with `--nextclade_ver`. Pangolin version used can be found in the generated report (and later be specified in the Dockerfile and then be run with `--keep_pangolin_ver` to run the pipeline without updating Pangolin) and Nextclade database version used can be found in the output folder `005_nextclade/nextclade-tag.json` (and later be specified with `--nextclade_data_ver`). Please note that the Nextclade version is currently not recorded, so it is recommended to set this with `--nextclade_ver` if staying in control of the version is important for your use case.
```

Comment:
The explanation about Pangolin in the parentheses is confusing to me. So the version is not yet specified in the `Dockerfile`? And the link to the flag `--keep_pangolin_ver` is that already implemented?
You wrote output folder for the Nextclad database version but specified a `json` file. Again you write about later, so it is not implemented yet?

Reply:
All the options are implemented. 
But I was trying to explain how to select the versions for later, if one wanted to go back and rerun a with the same Pangolin, Nextclade and Nextclade database versions.

To keep the commands backwards compatible Pangolin and Nextclade and its database is updated every run. I wanted to do it like this so that the people already using susCovONT, does not have to change the setup they are using if they implement our changes.

Right now the Pangolin version is set in the Dockerfile, but unless the `--keep_pangolin_ver` is specified it will get updated to the newest one.

I think my explanations probably needs more work to make this more understandable :/